### PR TITLE
Test columnMetadata empty response

### PR DESCRIPTION
**Pull request by michaljurecko**
Jira: https://keboola.atlassian.net/browse/PSGO-236
KBC: XXX

Before asking for review make sure that:

## Checklist

- [ ] New client method(s) has tests
- [ ] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Comment by michaljurecko** [comment]
(toto sa v CI netestuje?)

File: tests/Common/TablesListingTest.php [hunk 1]
```diff
@@ -186,6 +186,7 @@ public function testListTablesIncludeColumnMetadata(): void
         $this->assertArrayHasKey('columnMetadata', $firstTable);
         $this->assertArrayHasKey('metadata', $firstTable);
         $this->assertEmpty($firstTable['columnMetadata']);
+        $this->assertIsObject($firstTable['columnMetadata']);
```
Comment (jirkasemmler) [inline tests/Common/TablesListingTest.php:189]: ono to bude `[]` at uz se v response vrati cokoliv

Comment (tomasfejfar) [reply tests/Common/TablesListingTest.php:189]: Hmm, ještě je tam nějaký jiný problém...

```
[{"uri":"https:\/\/localhost:8700\/v2\/storage\/tables\/in.c-API-tests.languages","id":"in.c-API-tests.languages","name":"languages","displayName":"languages","transactional":false,"primaryKey":[],"indexType":null,"indexKey":[],"distributionType":null,"distributionKey":[],"syntheticPrimaryKeyEnabled":false,"created":"2023-06-30T11:00:37+0200","lastImportDate":"2023-06-30T11:00:46+0200","lastChangeDate":"2023-06-30T11:00:46+0200","rowsCount":5,"dataSizeBytes":1536,"isAlias":false,"isAliasable":true,"isTyped":false,"tableType":"table","metadata":[],"columnMetadata":[]}]
```

Comment (michaljurecko) [reply tests/Common/TablesListingTest.php:189]: Tak som asi skoncil :joy:

Comment (tomasfejfar) [reply tests/Common/TablesListingTest.php:189]: jj, zkusím to debugnout :(

**Comment by tomasfejfar** [comment]
Not easily testable (we'd need to test the exact response, not parse it from json)

**Pull request closed without merging**